ci/pipelines/aws-vault-template.yml

```diff
@@ -0,0 +1,204 @@
+meta:
+  docker-creds: &docker-creds
+    username: concourse
+    password: ((concourse_artifactory_password))
+  s3-params: &s3-params
+    bucket: ci-pix4d-concourse-pipeline
+    region_name: eu-west-1
+    access_key_id: ((concourse_user_access_key))
+    secret_access_key: ((concourse_user_secret_key))
+  release-params: &release-params
+    ARTIFACTORY_URL: https://artifactory.ci.pix4d.com/artifactory
+    ARTIFACTORY_USER: concourse
+    ARTIFACTORY_PASSWORD: ((concourse_artifactory_password))
+    CODESIGNER_TOKEN: ((codesigner_token))
+  status-handlers: &status-handlers
+    on_success:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: success}
+    on_failure:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: failure}
+    on_error:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: error}
+    on_abort:
+      put: cogito
+      no_get: true
+      inputs: [aws-vault.git]
+      params: {state: abort}
+
+##########################################################################################
+
+resource_types:
+  - name: cogito
+    type: registry-image
+    check_every: 24h
+    source:
+      repository: pix4d/cogito
+      tag: latest
+
+##########################################################################################
+
+resources:
+  - name: aws-vault.git
+    type: git
+    icon: git
+    webhook_token: ((concourse_gh_webhook))
+    source:
+      uri: git@github.com:pix4d/aws-vault.git
+      private_key: ((github_ssh_key))
+      branch: ((branch))
+      fetch_tags: true
+
+  - name: cogito
+    type: cogito
+    check_every: never
+    source:
+      owner: pix4d
+      repo: aws-vault
+      access_token: ((github_repo_status_token))
+      gchat_webhook: ((gchat_hook))
+
+  - name: archive-linux.s3
+    type: s3
+    icon: dump-truck
+    check_every: never
+    source:
+      regexp: artifacts/((project))/((branch))/aws-vault-v(\d+\.\d+\.\d+(-.+)*)-linux-amd64.tar.gz
+      <<: *s3-params
+
+  - name: archive-darwin-amd64.s3
+    type: s3
+    icon: dump-truck
+    check_every: never
+    source:
+      regexp: artifacts/((project))/((branch))/aws-vault-v(\d+\.\d+\.\d+(-.+)*)-darwin-amd64.zip
+      <<: *s3-params
```

Comment:
Do we really need to bother with this?

Reply:
> Do we really need to bother with this?

I have a feeling @odormond comment was more regarding the architecture.

Do we want to build for darwin amd64?

https://github.com/Pix4D/aws-vault/pull/3/files#r2285505453

Reply:
Ah, I misunderstood. Well, I am not sure. If the team says its useless then I can remove it...

Reply:
Yeah, I meant that we don't have to bother with intel macOS. I don't think we have any mac user in the company still using an intel mac.

Reply:
removed